src/components/widgets/minimap-widget/minimap-widget.js

```diff
@@ -1,8 +1,13 @@
 import { loadModules } from '@esri/react-arcgis';
 import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
 import MinimapWidgetComponent from './minimap-widget-component';
 import { disableInteractions, minimapLayerStyles, synchronizeWebScenes } from 'utils/minimap-utils';
 import HalfEarthModal from 'components/half-earth-modal/half-earth-modal';
+import actions from 'redux_modules/page-texts/page-texts';
```

Comment:
We do not need to import the `.js` file `'redux_modules/page-texts/page-texts'`, it will work with `'redux_modules/page-texts'`.
We are using the `DirectoryNamedWebpackPlugin` on the `config-overrides` so if the folder contains any file with the same name it will treat it as if it was the entry point :)